azerothcore/azerothcore-wotlk
# fix(Core/Spell): Make use of SPELL_ATTR7_IGNORE_COLD_WEATHER_FLYING

## Changes Proposed:
-  Make use of SPELL_ATTR7_IGNORE_COLD_WEATHER_FLYING
-  Affected spells:

```sql
SELECT `id`, `name_lang_1`, `attributesEx7`
FROM `dbc_spell` 
WHERE `attributesEx7` & @flag = @flag
```
| id | name_lang_1 | attributesEx7 | 
| ---: | --- | ---: | 
| 64681 | Loaned Gryphon | 4194304 | 
| 64749 | Loaned Gryphon | 4194304 | 
| 64761 | Loaned Wind Rider | 4194304 | 
| 64762 | Loaned Wind Rider | 4194304 | 
| 72422 | Quivering Strike | 4194304 | 
| 72453 | Quivering Strike | 4194304 | 



## Tests Performed:
- Built
- Tested in-game



## How to Test the Changes:

1. go on a char that does not have cold weather flying
2. .learn 64681
3. See you can mount that
4. .learn 32240
5. See you cannot mount that
6. .learn 54197
7. See you can mount both


## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
